### DISCUSSION TITLE
Retrieve child values from parent

### DISCUSSION THREAD

**Discussion by Vimalanathd** [dowjones/react-dropdown-tree-select#544] [Q&A]
Hi Team,

Would it be possible to get all child values upon selecting a parent in the search box? Please advise. Thanks

**Comment by mrchief** [comment]
It is. You'd need an HOC that:

 - keeps track of the original `data`
 - on parent select event, takes the parent's id
 - retrieves the child values from it's own data bag

Take a look at this example HOC to get started https://github.com/dowjones/react-dropdown-tree-select/blob/develop/docs/HOC.md#tree-node-paths
